week2/labs/lab2-3/public/styles/app.css

```diff
@@ -0,0 +1,104 @@
+body{
+  margin-left: 15px;
+  margin-top: 15px;
+  background: #0079bf;
+  font-family: helvetica neue;
+}
+
+h1 {
+  color: white;
+}
+
+.today-list{
+  background-color: #e2e4e6;
+  padding: 5px;
+  width: 300px;
+  border-radius: 3px;
+}
+
+.list-header{
+  padding-left: 9px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  font-weight: bold;
+  color: #4d4d4d;
+}
+
+.item{
+  height: 15px;
+  line-height: 15px;
+  margin: 5px;
+  padding: 7px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
+  background-color: white;
+  color: #4d4d4d;
+  border-radius: 3px;
+}
+
+.item-text-done{
+  font-size: 14px;
+  text-decoration: line-through;
+  color: #bbbbbb;
+}
+
+.item-text-undone{
+  font-size: 14px;
+}
+
+Button{
```

Comment:
HTML tags are case insensitive, but normally we try to stick with lowercase as a convention, so this is ebtter as `button`